packages/salesforcedx-vscode-apex/src/requirements.ts

```diff
@@ -99,7 +99,7 @@ function isLocal(javaHome: string): boolean {
   return !path.isAbsolute(javaHome);
 }
 
-function checkJavaVersion(javaHome: string): Promise<any> {
+export function checkJavaVersion(javaHome: string): Promise<any> {
```

Comment:
should the return type here be `Promise<void>`?

Reply:
or actually maybe `Promise<boolean>` if we want to match the `resolve(true)` I'd probably lean towards updating that to be just `resolve();` and the void return type if we aren't using the result.

Reply:
Done, it's now typed as returning `Promise<boolean>`